fundrebalancer/fundrebalancer.go

```diff
@@ -472,6 +480,16 @@ func (r *FundRebalancer) GetRebalanceTxns(
 				return nil, fmt.Errorf("hex decoding evm call data: %w", err)
 			}
 
+			if err := r.ERC20Approval(ctx, SkipGoTxnWithMetadata{
+				tx:                 skipgo.Tx{EVMTx: txn.EVMTx},
+				sourceChainID:      sourceChainID,
+				destinationChainID: destChainID,
+				amount:             amount,
+			}); err != nil {
+				return nil, fmt.Errorf("handling ERC20 approval: %w", err)
+			}
+			lmt.Logger(ctx).Info("", zap.Any("sourceChainID", sourceChainID), zap.Any("destChainID", destChainID), zap.Any("amount", amount))
```

Comment:
possibly remove, maybe this was for debugging? seems kinda unnecessary to log

Reply:
yep forgot to remove